assets/js/prettyPhoto/jquery.prettyPhoto.init.js

```diff
@@ -2,15 +2,7 @@
 $(function() {
 
 	// Lightbox
-	$("a.zoom").prettyPhoto({
-		hook: 'data-rel',
-		social_tools: false,
-		theme: 'pp_woocommerce',
-		horizontal_padding: 20,
-		opacity: 0.8,
-		deeplinking: false
-	});
```

Comment:
The typo changes look good. Thanks for those. What is the reasoning behind this change?

Reply:
Separate PR for unrelated changes next time please.